index.php

```diff
@@ -19,9 +24,53 @@
 
             <input type="search" class="input-search" placeholder="Search Here">
 
+
         </header>
+
+        <section class="content">
+
+            <nav>
+                <ul class="navbar">
+                    <li><a href="#about">About</a></li>
+                    <li><a href="#seasonal" class="active">Seasonal</a></li>
+                    <li><a href="#category">Category</a></li>
+
+                </ul>
+            </nav>
+
+            <div class="fruit-definition">
+                <h1>Pineapple</h1>
+                <p>The pineapple is a tropical plant with an edible fruit, also called a pineapple and the most economically significant plant in the family Bromeliaceae.</p>
+
+            </div>
+
+            <div class="center_image">
+                <img src="images/pineapple.png" alt="pineapple">
+            </div>
+
+            <div class="fruit-facts">
+
+                <div class="health-benefits">
+                    <img class="straight_line" src="images/straight_line.png" alt="straight red line">
+                    <h2>Health Benefits</h2>
+                    <p>Pineapple contains significant amount of vitamin C. a water-soluble antioxidant that fights cell damage.</p>
+                </div>
+
+                <div class="nutrion-values">
+                    <img class="l_line" src="images/l_line.png" alt="L line">
```

Comment:
Same comment above:

>This class is also using _ instead of -|

Also, use a more descriptive name for the image and the class